downloader/artifact_downloader.go

```diff
@@ -0,0 +1,111 @@
+package downloader
+
+import (
+	"io"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+const DOWNLOAD_PATH = "_tmp" // under current work dir
+const FILE_PERMISSION = 0755
+const MAX_CONCURRENT_DOWNLOAD_THREAD = 10
```

Comment:
All constants should be camelCase. Also, const values can be grouped together in one const declaration.